modules/chapter2/pages/rbac.adoc

```diff
@@ -0,0 +1,138 @@
++
+image::rbac-configmap.png[title=RBAC ConfigMap Configuration]
+
+. Re-deploy the helm chart to re-read the new configuration.
+
+. Sign out and sign in again as the user (the user who was assigned the `admins` role) you mentioned in the policy CSV file. Navigate to the `Catalog` page, and click `CREATE`. Note that you are now allowed to create new components.
+
+. Click `API` and note that you are now allowed to register new APIs.
+
+. Try and register a new API or component by entering `https://github.com/backstage/backstage/blob/master/catalog-info.yaml` into the `Select URL` field, and then click `ANALYZE`. Note that the *deny* policy for `catalog.location` in your RBAC prevents RHDH from fetching the metadata about the component. You will see an error:
++
+[subs=+quotes]
+----
+{"error":{"name":"NotAllowedError","message":""},"request":{"method":"POST","url":"/locations?dryRun=true"},"response":{"statusCode":403}}
+----
+
+. Change the `rhdh-rbac-policy` ConfigMap to *allow* `create` and `read` actions for the `catalog.location` resource and re-deploy your helm chart. You should now be able to create new components and the metadata fetches from remote Git repositories should work as before.
+
+. You can experiment with enabling and disabling various components of RHDH by following the permissions guide at https://github.com/janus-idp/backstage-plugins/blob/main/plugins/rbac-backend/docs/permissions.md. Some plugins and their features can be controlled using RBAC.
+
+WARNING: There is a bug in the RBAC system when database persistence is enabled for RBAC. Do not enable the `database` attribute in the rbac `permission` block in `app-config-rhdh`!
+
+WARNING: You may encounter inconsistencies and bugs with RBAC, and policy examples are scarcely documented. There are plans to introduce a web based UI for policy management in future releases of RHDH. It is recommended to keep the RBAC system *disabled* when you trying out features and functionality of RHDH like Software Templates, plugins, Search, and more. Turn it back on only if you know what you are doing and you fully understand the impact of policy files.
+
+## RBAC REST API
+
+RHDH provides an RBAC REST API that you can use to manage the permissions and roles programmatically. This API can be used to automate the maintenance of RHDH permission policies and roles.
+
+You can perform the following actions with the REST API:
+
+* Retrieve information about all permission policies or specific permission policies, or roles
+* Create, update, or delete a permission policy or a role
+* Retrieve permission policy information about static plugins
+
+WARNING: If RBAC is enabled, you may need to permit actions on the `policy.entity` resource for read, create and update to view and change policies using the REST API. Recall that default policy is to deny, so you will see `HTTP 401 Unauthorized` errors when you make REST calls without policy changes.
```

Comment:
From what I saw is that in such cases you end up with 403 Forbidden rather than 401 Unauthorized

Reply:
fixed